pom.xml

```diff
@@ -41,12 +41,13 @@
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-			<flink.version>0.10.1</flink.version>
-			<!-- Default parameters for mvn exec:exec -->
-			<clazz>com.dataartisans.flink.dataflow.examples.WordCount</clazz>
-			<input>kinglear.txt</input>
-			<output>wordcounts.txt</output>
-			<parallelism>1</parallelism>
+		<flink.version>1.0-SNAPSHOT</flink.version>
+		<beam.version>1.5.0-SNAPSHOT</beam.version>
+		<!-- Default parameters for mvn exec:exec -->
+		<clazz>com.dataartisans.flink.dataflow.examples.WordCount</clazz>
+		<input>kinglear.txt</input>
+		<output>wordcounts.txt</output>
+		<parallelism>1</parallelism>
```

Comment:
Sorry about that - I started using intellij with space instead of tabs and all hell broke loose. Clearly didn't get it put back together.